src/traces/indicator/plot.js

```diff
@@ -650,11 +622,46 @@ function drawNumbers(gd, plotGroup, cd, opts) {
                 return deltaFormatText(deltaValue(cd[0]));
             });
         }
+
+        delta.attr('data-unformatted', deltaFormatText(deltaValue(cd[0])));
```

Comment:
I'm curious. Why did you use `data-unformatted` here?

Reply:
Because `Drawing.bBox` fails without it when it tries to hash the node:
https://github.com/plotly/plotly.js/blob/44633a13a3e022a4565d7b230b9db3e1e1ac83aa/src/components/drawing/index.js#L990-L991

Reply:
Cool. Thanks for the info! Nice find!